### PR TITLE
feat(arc-1240): add image grid option: 30 percent wide

### PR DESCRIPTION
**Pull request by bertyhell**
https://meemoo.atlassian.net/browse/ARC-1240

client PR: https://github.com/viaacode/hetarchief-client/pull/994

https://github.com/viaacode/react-admin-core-module/assets/1710840/5e307820-756f-475b-a7b3-118520efc12e

### CODE REVIEW

**Review by SHDBramVandenbussche — APPROVED** [review]

File: ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockImageGrid.tsx [hunk 1]
```diff
@@ -8,9 +8,9 @@ import './BlockImageGrid.scss';
 
 export const BlockImageGrid: FunctionComponent<BlockImageGridProps> = ({
 	elements = [],
-	imageWidth = 200,
-	imageHeight = 200,
-	itemWidth = 200,
+	imageWidth = '200px',
```
Comment (WardVerc1) [inline ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockImageGrid.tsx:11]: use rem

File: ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockImageGrid.wrapper.tsx [hunk 2]
```diff
@@ -18,23 +20,30 @@ export const formatLookup: {
...
 	};
 } = {
-	squareSmall: { imageWidth: 200, imageHeight: 200, itemWidth: 200 },
-	squareLarge: { imageWidth: 275, imageHeight: 275, itemWidth: 275 },
-	'4:3': { imageWidth: 400, imageHeight: 300, itemWidth: 400 },
-	'2:1': { imageWidth: 200, imageHeight: 100, itemWidth: 200 },
-	'6:9': { imageWidth: 400, imageHeight: 225, itemWidth: 400 },
-	'400x150': { imageWidth: 400, imageHeight: 150, itemWidth: 400 },
-	'384x220': { imageWidth: 384, imageHeight: 220, itemWidth: 384 },
+	flex3: { imageWidth: '100%', imageHeight: '220px', itemWidth: '30%' },
```
Comment (WardVerc1) [inline ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockImageGrid.wrapper.tsx:28]: use rem

File: ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockLogoGrid.editorconfig.ts [hunk 3]
```diff
@@ -40,7 +40,7 @@ export const INITIAL_LOGO_GRID_BLOCK_STATE = (): BlockLogoGridWrapperProps &
 		format: '2:1',
 		fill: 'cover',
 		textAlign: 'center',
-		itemWidth: 400,
+		itemWidth: '400px',
```
Comment (WardVerc1) [inline ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockLogoGrid.editorconfig.ts:43]: use rem

File: ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockLogoGrid.wrapper.tsx [hunk 4]
```diff
@@ -18,7 +18,7 @@ export interface BlockLogoGridWrapperProps {
 
 export const BlockLogoGridWrapper: FunctionComponent<BlockLogoGridWrapperProps> = ({
 	format = '2:1',
-	itemWidth = 400,
+	itemWidth = '400px',
```
Comment (WardVerc1) [inline ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockLogoGrid.wrapper.tsx:21]: use rem

File: ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockUspGrid.wrapper.tsx [hunk 5]
```diff
@@ -14,9 +14,9 @@ export const BlockUspGridWrapper: FunctionComponent<BlockUspGridWrapperProps> =
 	return (
 		<div style={{ width: 'calc(100% + 124px)', marginLeft: '-62px' }}>
 			<BlockImageGrid
-				imageWidth={325}
-				imageHeight={75}
-				itemWidth={325}
+				imageWidth="325px"
```
Comment (WardVerc1) [inline ui/src/react-admin/modules/content-page/components/blocks/BlockImageGrid/BlockUspGrid.wrapper.tsx:17]: use rem

**Review by WardVerc1 — APPROVED** [review]

**Pull request merged**